cmd/serve/httplib/serve/data/data.go

```diff
@@ -0,0 +1,40 @@
+//go:generate go run assets_generate.go
+// The "go:generate" directive compiles static assets by running assets_generate.go
+
+package data
+
+import (
+	"html/template"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+)
+
+// GetTemplate eturns the HTML template for serving directories via HTTP
+func GetTemplate() (tpl *template.Template, e error) {
+	tpl = nil
```

Comment:
You can rely on variables (including return parameters) being initialized to zero, so these two lines are unnecessary